src/apiSchemas/request/schemaValidator.ts

```diff
@@ -0,0 +1,37 @@
+import { ObjectSchema } from 'joi';
+import { Request, Response, NextFunction } from 'express';
+// eslint-disable-next-line import/no-unresolved
+import { Query } from 'express-serve-static-core'; // todo: to resolve
+import schemas from './validators/index';
+
+type RequestHandler = (req: Request, res: Response, next: NextFunction) => void;
+
+const validateRequestSchema = (schema: string): RequestHandler => {
+  const selectedSchema: ObjectSchema = schemas[schema];
+
+  if (!selectedSchema) {
+    throw new Error(`${schema} does not exist`);
```

Comment:
As I understand this Error occurs when the project starts if schema is not found. Could you consider covering this case in tests, or passing a schema object instead of the schema name?